tests/puppeteer/campaigns/functional/BO/11_international/01_localization/languages/01_filterAndQuickEditLanguages.js

```diff
@@ -169,4 +173,64 @@ describe('Filter Languages', async () => {
       });
     });
   });
+
+  describe('Disable default language', async () => {
+    it('should filter by iso_code \'en\'', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterToDisableDefaultLanguage', baseContext);
+      await this.pageObjects.languagesPage.filterTable('input', 'iso_code', Languages.english.isoCode);
+      const numberOfLanguagesAfterFilter = await this.pageObjects.languagesPage.getNumberOfElementInGrid();
+      await expect(numberOfLanguagesAfterFilter).to.be.at.least(1);
+      const textColumn = await this.pageObjects.languagesPage.getTextColumnFromTable(1, 'iso_code');
+      await expect(textColumn).to.contains(Languages.english.isoCode);
+    });
+
+    it('should disable \'en\' language and check error message', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'disableMainLanguage', baseContext);
+      await this.pageObjects.languagesPage.quiEditLanguage(1, false);
+      const textError = await this.pageObjects.languagesPage.getAlertDangerMessage();
+      await expect(textError).to.equal(this.pageObjects.languagesPage.unSuccessfulUpdateDefaultLanguageStatusMessage);
+    });
+
+    it('should reset all filters', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'disableDefaultLanguageReset', baseContext);
+      const numberOfLanguagesAfterReset = await this.pageObjects.languagesPage.resetAndGetNumberOfLines();
+      await expect(numberOfLanguagesAfterReset).to.equal(numberOfLanguages);
+    });
+  });
+
+  describe('Quick edit language', async () => {
+    it('should filter by iso_code \'fr\'', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'filterToQuickEdit', baseContext);
+      await this.pageObjects.languagesPage.filterTable('input', 'iso_code', Languages.french.isoCode);
+      const numberOfLanguagesAfterFilter = await this.pageObjects.languagesPage.getNumberOfElementInGrid();
+      await expect(numberOfLanguagesAfterFilter).to.be.at.least(1);
+      const textColumn = await this.pageObjects.languagesPage.getTextColumnFromTable(1, 'iso_code');
+      await expect(textColumn).to.contains(Languages.french.isoCode);
+    });
+
+    const tests = [
+      {args: {action: 'disable', enabledValue: false}},
+      {args: {action: 'enable', enabledValue: true}},
+    ];
+    tests.forEach((test) => {
+      it(`should ${test.args.action} first language`, async function () {
+        await testContext.addContextItem(this, 'testIdentifier', `${test.args.action}Language`, baseContext);
+        const isActionPerformed = await this.pageObjects.languagesPage.quiEditLanguage(1, test.args.enabledValue);
```

Comment:
```suggestion
        const isActionPerformed = await this.pageObjects.languagesPage.quickEditLanguage(1, test.args.enabledValue);
```

Reply:
Not sure this is the only place you need to fix.

Reply:
I fixed it in 2 places :+1: (all the file)